.github/workflows/_release_image.yml

```diff
@@ -0,0 +1,53 @@
+name: Release Individual Image
+
+on:
+  workflow_call:
+    inputs:
+      repo_type:
+        description: "Type of repository (public/private)"
+        required: true
+        type: string
+      image:
+        description: "Image to push"
+        required: true
+        type: string
+      tag:
+        description: "Tag to push the image with"
+        required: true
+        type: string
+
+jobs:
+  get_docker_context:
+    name: Get Docker Context
+    runs-on: ubuntu-latest
+    outputs:
+      docker_context: ${{ steps.get_docker_context.outputs.docker_context }}
+      dockerfile_path: ${{ steps.get_docker_context.outputs.dockerfile_path }}
+    steps:
+      - name: Get Docker Context and Dockerfile Path
+        id: get_docker_context
+        run: |
+          if [[ "${{ inputs.image }}" == "aci/helloworld" ]]; then
+            context="hello-world"
+            dockerfile_path="hello-world/ACI"
+          elif [[ "${{ inputs.image }}" == "aks/helloworld" ]]; then
+            context="hello-world"
+            dockerfile_path="hello-world/AKS"
+          else
+            context="."
```

Comment:
when does this `else` happen? If it's not a normal scenario, maybe we should just fail out

Reply:
It's for the kafka files. They're built from the root dir